### PR TITLE
Release notes post for 0.4.1 

### PR DESCRIPTION
**Pull request by derberg**
**Description**

Changes proposed in this pull request:

- Release notes post for 0.4.1

### CODE REVIEW

**Comment by derberg** [comment]
@magicmatatjahu hey man, CI is failing because I have relative links here that of course do not work. What is the best to change in milv config to fix CI? I mean how can I easily provide a base domain for relative links?

**Comment by magicmatatjahu** [comment]
Hi @derberg I only see one solution and you won't like it: add all relative links to white list. Also I could enable configuration for regex in milv.config.yaml, because at the moment relative links must be set with whole value.
Providing base path for relative links is bad idea, because base path is already as option in config of milv and it is used for setting absolute path for checking exist files in OS or volume in container. About other problems with this, we can talk tomorrow morning.

**Comment by derberg** [comment]
@magicmatatjahu yeap we need to talk and you need to convince me hard because awesome-bot supports what I need `--base-url [base url]        Base URL to use for relative links` and it is one of the most important links validation feature

**Comment by derberg** [comment]
@magicmatatjahu ok you are right, as long as we are not done with fix for this bug https://github.com/kyma-project/website/issues/34 I have to do the whitelisting of those paths

File: src/blog-posts/5-release-notes-041.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+The Service Catalog that is one of out key features got a lot of improvements over last months, mainly thanks to your valuable feedback. To be more specific, Console UI part of the Catalog was improved:
+- Multistep service provisioning wizard is now replaced with one step wizard
+- In list of ServiceInstances and the ServiceInstance details now you can preview the instance parameters by clicking the instance Plan name
+![](./assets/instance_params.png)
+- Individual left navigation is removed and filtering moved next to the search. Now the filtering is enabled to support more different filters in the future and the scales much better in case there are many values provided in the filters.
+![](./assets/filter.png)
+
+Console UI is not everything though. The following things are done as well:
+- Catalog is upgraded to v0.1.34 with namespace-scope resources (ServiceBroker, ServiceClass, ServicePlan) support. Related Console views do not support it yet.
+- In Grafana you can find a dedicated Service Catalog dashboard that improves operations
```
[the remark this answers is not in the document]
Comment (derberg) [reply src/blog-posts/5-release-notes-041.md]: just did `provided` and not `created

File: src/blog-posts/5-release-notes-041.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+## Application Connector
+
+The Application Connector, that alows you to connect external systems into Kyma went through major improvements:
+- Management of the Remote Environments (RE) is no longer done through the Helm chart. Now we have a controller that reacts on changes in RE and setups the whole environment,
+- The [RemoteEnvironments CRD](/docs/latest/components/application-connector#custom-resource-remoteenvironment) is now enhanced with extra label field. Now you can now in a flexible way mark what is the purpose of the Remote Environment,
+- The Application Connector has been powered by OAuth token caching functionality.
+
+## Logging 
+
+Kyma has not a new component to enhance its logging capabilities. It uses [Logspout and OK Log](/docs/latest/components/logging).
```
[the remark this answers is not in the document]
Comment (derberg) [reply src/blog-posts/5-release-notes-041.md]: replaced `uses` with `is based on`

**Pull request merged**